proxy/ptunnel/commands/agent.go

```diff
@@ -46,5 +52,18 @@ func RunAgentCommand(bf *client.BaseFlags, config *agentConfig) error {
 			fmt.Printf("PKS: %s Identities: %v ValidFor: %s \n", p.MD5, p.Principals, p.ValidFor.String())
 		}
 	}
+	if len(config.RemainingArgs) >= 1 {
+		cmd := exec.Command("sh", append([]string{"-c"}, strings.Join(config.RemainingArgs[:], " "))...)
+		cmd.Stdout = command.OutOrStdout()
+		cmd.Stderr = command.ErrOrStderr()
+		cmd.Stdin = command.InOrStdin()
+		cmd.Env = agent.GetEnv()
+		if config.UseHostEnv {
+			cmd.Env = append(os.Environ(), cmd.Env...)
+		}
+		if err := cmd.Run(); err != nil {
+			return err
```

Comment:
ah! forgot. When you run a command, `enkit agent` should exit with the status of that command. Eg, if the command fails with exit status 5, `enkit agent command` should also exit with that status.

You can achieve this with something like:
```
    var exit *exec.ExitError
    if errors.As(err, &exit) {
        return kflags.NewStatusError(exit.ExitCode(), err)
    }
```
(code not checked, don't remember if it should be a pointer or not in the var)

Reply:
Done, and nice catch.